src/main/java/net/mcreator/minecraft/link/CurrentDevice.java

```diff
@@ -147,14 +147,23 @@ public static void disableDigitalInputEvents(int port) {
 	 * <p>
 	 * If there is no device connected, nothing happens.
 	 *
-	 * @param message The message to be sent
+	 * @param command The message to be sent
 	 */
-	public static void sendMessage(String message) {
+	public static void sendMessage(String command) {
 		AbstractDevice device = MCreatorLink.LINK.getConnectedDevice();
+		System.out.println("pure message: " + command);
 		if (device != null)
-			device.sendData(("msg?" + message + "\n").getBytes());
+			device.sendData(("msg?" + command + "\n").getBytes());
 	}
 
+	public static void sendMessage(String command, String data){
```

Comment:
Add javadoc here

Reply:
Quick note, I did it manually, because my JavaDoc generated had different files and Polish language and also contained a bunch of "index-#.html" files so I just checked what needed to be replaced and I did it.

Reply:
Docs will be regenerated, you only need it here, so you should revert javadocs file

Reply:
Okay, reverting changes in /docs